rpc_util.go

```diff
@@ -164,7 +164,7 @@ func encode(c Codec, msg interface{}, pf payloadFormat) ([]byte, error) {
 	// Write message into the fixed header.
 	buf.WriteByte(uint8(pf))
 	var b []byte
-	var length uint32
+	var length uint
```

Comment:
I guess we're already covered on 32-bit machines, since you can't have a too-big []byte there. So the only thing to worry about is 64-bit machines. We should probably have a test for too-long messages.  ;-)

I don't think we need to care about supporting Go 1.3. That's ancient at this point. Go 1.4 is the only baseline, and it's what the grpc-go README says too.